docs/tutorials-be-gone.rst

```diff
@@ -0,0 +1,29 @@
+tutorials-be-gone
```

Comment:
Suggestion from discord: hide-tutorials. More consistent with existing tool names and hopefully easier to guess. I do like the creative part of the name though

Reply:
done